### PR TITLE
HV-115 - Changes missing in the original commit.

### PR DESCRIPTION
**Pull request by mdone-cloudian**
Some field changes were not comited and present on the original PR.

### CODE REVIEW

File: build.go [hunk 1]
```diff
@@ -360,10 +360,15 @@ func createPackage(options linuxPackageOptions) {
 
 	args := []string{
 		"-s", "dir",
-		"--description", "Grafana",
+		// cloudian customization
+		"--description", "Grafana with some customizations.",
```
Comment (gbunt) [inline build.go:364]: "Cloudian fork of Grafana" maybe?

File: build.go [hunk 2]
```diff
@@ -360,10 +360,15 @@ func createPackage(options linuxPackageOptions) {
...
+
 		"-C", packageRoot,
-		"--url", "https://grafana.com",
-		"--maintainer", "contact@grafana.com",
+
+		// cloudian customization
+		"--url", "https://cloudian.com",
+		"--maintainer", "Cloudian Engineering <cloudian-eng@cloudian.com>",
```
Comment (gbunt) [inline build.go:370]: This is an internal mailinglist email address. We should think what email address we want to use as maintainer email. If we don't want anyone to contact us, we may as well leave it out.

Comment (gbunt) [reply build.go:370]: @snaik1 any suggestions on what email address would make sense here?

Comment (mdone-cloudian) [reply build.go:370]: Maybe we can create a `packaging@cloudian.com`, but who would read it?

Comment (mdone-cloudian) [reply build.go:370]: Antonette is going to create it.

**Pull request merged**